components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt

```diff
@@ -165,6 +172,18 @@ data class TopFrecentSiteInfo(
     val title: String?
 )
 
+/**
+ * Frecency threshold options for fetching top frecent sites. Requests a page that was visited
+ * with a frecency score greater or equal to the [value].
+ */
+enum class FrecencyThresholdOption(val value: Long) {
```

Comment:
Do you anticipate this growing over time? It could, in theory... but if not, a simple flag would have been simpler.

Reply:
Only in theory. I am accounting for potential changes, but don't currently anticipate anything.

Realistically, this could've been just a value (101) that we feed into the AS API or hardcoded somewhere, but I didn't want to expose a magical number like 101 which might not signify anything to consumers.